src/sympc/protocol/falcon/falcon.py

```diff
@@ -0,0 +1,33 @@
+"""FALCON Protocol.
+
+FALCON : Honest-Majority Maliciously Secure Framework for Private Deep Learning.
+arXiv:2004.02229 [cs.CR]
+"""
+# stdlib
+from typing import Any
+from typing import Dict
+from typing import List
+
+from sympc.protocol.protocol import Protocol
+from sympc.tensor import ReplicatedSharedTensor
+from sympc.tensor.tensor import SyMPCTensor
+
+
+class FALCON(metaclass=Protocol):
```

Comment:
NITPICK: Could you make it snake case? (`Falcon`) - FSS looks like that because it is a short form from Function Secret Sharing (also for the Protocol name)

Reply:
Sure @gmuraru